sigle/src/modules/profileCard/ProfileCard.tsx

```diff
@@ -185,18 +185,40 @@ export const ProfileCard = ({
           {settings.siteDescription}
         </Typography>
         <Flex gap="3">
-          <Typography size="subheading" css={{ color: '$gray9' }}>
-            <Box css={{ color: '$gray11', mr: 2 }} as="span">
-              {followers ? followers?.length : 0}
-            </Box>
-            Followers
-          </Typography>
-          <Typography size="subheading" css={{ color: '$gray9' }}>
-            <Box css={{ color: '$gray11', mr: 2 }} as="span">
-              {following ? following?.length : 0}
-            </Box>
-            Following
-          </Typography>
+          <Link
```

Comment:
For this file as the API is now returning the count for the following / followers this file can be optimised and we can remove the useGetUsersFollowing hook and just use the user info. This would save us 2 API calls and data will load faster :)